src/variorum/IBM/config_ibm.c

```diff
@@ -26,7 +26,7 @@ int set_ibm_func_ptrs(void)
     if (*g_platform.ibm_arch == POWER9)
     {
         g_platform.variorum_print_power = p9_get_power;
-        g_platform.variorum_print_power_limits = p9_get_power_limits;
+        g_platform.variorum_print_power_limit = p9_get_power_limits;
```

Comment:
Align equal signs, might need to bump to the right

Reply:
Alignment not needed here as surrounding lines are long and broken into two lines